code/modules/organs/blood.dm

```diff
@@ -7,7 +7,8 @@ var/const/BLOOD_VOLUME_OKAY =    75
 var/const/BLOOD_VOLUME_BAD =     60
 var/const/BLOOD_VOLUME_SURVIVE = 40
 
-/mob/living/carbon/human/var/datum/reagents/vessel // Container for blood and BLOOD ONLY. Do not transfer other chems here.
+/mob/living/carbon
+	var/datum/reagents/vessel // Container for blood and BLOOD ONLY. Do not transfer other chems here.
 /mob/living/carbon/human/var/var/pale = 0          // Should affect how mob sprite is drawn, but currently doesn't.
```

Comment:
You edited the line above but this one has the exact same syntax. Why the change?

Reply:
im not really sure this is belong here instead of carbon/human define, but still this way its more distinct and constant